packages/chat-core-zendesk/src/infra/ChatCoreZendeskImpl.ts

```diff
@@ -210,4 +217,10 @@ export class ChatCoreZendeskImpl {
   resetSession(): void {
     this.conversationId = undefined;
```

Comment:
I didn't include it in the first pass of this package, but I think it would be good to add `Smooch.off();` here to remove all event listeners attached to old conversation  (note that the typing might not have support for this so you can just do `@ts-ignore` but it should be a thing: https://github.com/zendesk/sunshine-conversations-web?tab=readme-ov-file#events)

Reply:
Yep! Good call